cfme/intelligence/chargeback.py

```diff
@@ -239,3 +251,34 @@ def delete(self):
         tb_select('Remove from the VMDB', invokes_alert=True)
         sel.handle_alert()
         flash.assert_no_errors()
+
+
+class Assign(Updateable, Pretty):
+
+    def __init__(self, assign_to=None,
+                 tag_category=None,
+                 selections=None
+                 ):
+        self.assign_to = assign_to
+        self.tag_category = tag_category
+        self.selections = selections
+
+    def storageassign(self):
+        sel.force_navigate('chargeback_assignments_storage', context={'chargeback': self})
+        fill(assign_form,
+        dict(
+            {'assign_to': self.assign_to,
+             'tag_category': self.tag_category,
+             'selections': self.selections}),
+            action=assign_form.save_button)
```

Comment:
This is making a dict of a dict, so the `dict()` wrapper is not needed.

`assert dict({1:2, 3:4, 5:6}) == {1:2, 3:4, 5:6}`